lib/godmin/engine_wrapper.rb

```diff
@@ -17,7 +17,7 @@ def namespaced?
     def namespaced_path
       @namespaced_path ||= begin
         if namespaced?
-          namespace.name.classify.split("::").map(&:underscore)
+          namespace.name.split("::").map(&:underscore)
```

Comment:
I guess it depends on what `engine.railtie_namespace` returns.. Or `Rails::Generators.namespace` for that matter in `generators/base` if we want to keep them synced for some reason.